### DISCUSSION TITLE
"Uncloak canonical names" missing in settings

### DISCUSSION THREAD

**Discussion by Katzenwerfer** [dhowe/AdNauseam#2530] [Q&A]
uBlock Origin has a feature to do CNAME-uncloaking to identify more 3rd-party servers.
I noticed this was nowhere to be found in the settings interface, unlike with uBlock, though I also tried to glance at the extensions code and there are some references to said functionality.
Thought there might be a reason behind this. So, why is this the case?

**Comment by dhowe** [comment]
I also don't see this in the ublock settings page for adn:
@mneunomne  ?
<img width="631" alt="image" src="https://github.com/dhowe/AdNauseam/assets/737638/b03a24e2-9dae-4dfd-85e0-83b31fb5e100">
